yt_dlp_plugins/postprocessor/stash.py

```diff
@@ -72,4 +79,214 @@ def run(self, info):
+
+            # Step to handle performers
+            if scene_data.get("performers"):
+                performer_ids = []
+                for performer in scene_data["performers"]:
+                    performer_name = performer["name"]
+                    performer_url = performer.get("url")
+                    
+                    # Search for existing performer
+                    existing_performer = self.stash.find_performers({"name": {"modifier": "EQUALS", "value": performer_name}})
+                    
+                    if existing_performer and len(existing_performer) > 0:
+                        performer_ids.append(existing_performer[0]["id"])
+                    else:
+                        # Create the performer if they don't exist
+                        new_performer = {"name": performer_name}
+                        if performer_url:
+                            new_performer["url"] = performer_url
+                        
+                        created_performer = self.stash.create_performer(new_performer)
+                        performer_ids.append(created_performer["id"])
+                
+                # Add performer IDs to the update payload
+                update_scene["performer_ids"] = performer_ids
+
+            if scene_data.get("studio"):
+                studio_name = scene_data["studio"]["name"]
+                existing_studio = self.stash.find_studios({"name": {"modifier": "EQUALS", "value": studio_name}})
+                
+                if not existing_studio:
+                    existing_studio = self.stash.find_studios({"aliases": {"modifier": "EQUALS", "value": studio_name}})
+                
+                if existing_studio and len(existing_studio) > 0:
+                    update_scene["studio_id"] = existing_studio[0]["id"]
+                    self.to_screen(f"[Debug] using existing studio {existing_studio}")
+                else:
+                    # Create the studio if it doesn't exist
+                    self.to_screen("[Debug] creating new studio")
+                    new_studio = {
+                        "name": studio_name,
+                        "url": scene_data["studio"]["url"] if scene_data["studio"].get("url") else None
+                    }
+                    created_studio = self.stash.create_studio(new_studio)
+                    update_scene["studio_id"] = created_studio["id"]
+
+            self.to_screen(f"[Debug] Scene update payload: {update_scene}")
+
+            try:
+                self.stash.update_scene(update_scene)
+                self.to_screen("[Debug] Scene updated with scraped metadata.")
+            except Exception as e:
+                self.to_screen(f"[Error] Error updating scene metadata: {str(e)}")
+
+        except Exception as e:
+            self.to_screen(f"[Error] Unexpected error during processing: {str(e)}")
+
+        return [], info
+    
+    def scrape_scene_by_url(self, url):

```

Comment:
```suggestion
    def _scrape_scene_by_url(self, url):

```
I like to put _ before internal functions.